engine/src/main/java/org/terasology/world/chunks/CoreChunk.java

```diff
@@ -153,6 +154,50 @@
      * @return Liquid state currently assigned to the block
      */
     LiquidData getLiquid(int x, int y, int z);
+    
+    /**
+     * Sets one of the per-block custom data values at a given position relative to the chunk.
+     * The given value is downcast from int to the appropriate type for the array. It is not
+     * checked for overflow.
```

Comment:
It's not clear what you're saying, is there a typo?

In most cases if someone attempts to set a value outside of the range possible for the array type, this would probably not be too surprising a result, I just thought it was worth documenting exactly what the behaviour was.

Reply:
"It is not checked for overflow." just stuck out to me and made me wonder what it could lead to :-) Good to have it documented, if you're not worried about it otherwise I won't either